packages/@okta/vuepress-site/docs/guides/configure-access-policy/main/index.md

```diff
@@ -0,0 +1,222 @@
+---
+title: Configure an access policy
+meta:
+  - name: description
+    content: Okta access policies help you secure your APIs by defining different access and refresh token lifetimes for a given combination of grant type, user, and scope.
+layout: Guides
+---
+
+This guide helps you understand access policies and how to configure them for common scenarios.
+
+---
+<!-- Nutrition facts bullets -->
+**Learning outcomes**
+
+* Understand access policies.
+* Configure an access policy to limit which scopes some clients can access.
+* Configure a custom access token lifetime per client.
+
+**What you need**
+
+* [Okta Developer Edition organization](https://developer.okta.com/signup)
+* An OIDC client app for testing access policies
+
+<ApiAmProdWarning />
+
+---
+
+## Overview
```

Comment:
It's not in the Confluence doc, but I think that Christine prefers that we avoid using "Overview" here. Something like "About configuring access policies" could be more informative.